mirror/github/clone_repos.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import click
+import pygit2
```

Comment:
Add to setup.py as a dependency.

Reply:
@Andrei-Dolgolev : Let us get rid of `requirements.txt` and only use `setup.py`. Make sure that all the required dependencies are present in `setup.py`.